editors/vscode/src/common.ts

```diff
@@ -132,21 +134,43 @@ export function languageClientOptions(
 
 export function activate(
     context: vscode.ExtensionContext,
-    client: ClientHandle,
-    startClient: (_ctx: vscode.ExtensionContext) => void,
+    startClient: (_client: ClientHandle, _ctx: vscode.ExtensionContext) => void,
 ): [vscode.StatusBarItem, PropertiesViewProvider] {
     const statusBar = vscode.window.createStatusBarItem(
         vscode.StatusBarAlignment.Left,
     );
-    context.subscriptions.push(statusBar);
-    statusBar.text = "Slint";
-
-    startClient(context);
 
     const properties_provider = new PropertiesViewProvider(
         context.extensionUri,
     );
 
+    context.subscriptions.push(statusBar);
+    statusBar.text = "Slint";
+
+    client.add_updater((cl) => {
+        if (cl !== null) {
+            cl.registerFeature(new snippets.SnippetTextEditFeature());
```

Comment:
Moving the snippet feature registration here caused the "Wrap in element" and "Remove element" code actions to go missing. I think the static feature needs to be registered before starting the client. Would it be ok to move this either back to `browser.ts` and `extension.ts` or add some other helper here that gets called before starting the client?